rbc/heavydb/row_function_manager.py

```diff
@@ -0,0 +1,132 @@
+__all__ = ['HeavyDBRowFunctionManagerType']
+
+
+from numba.core import extending, cgutils
+from numba.core.cgutils import make_bytearray, global_constant
+from numba.core import types as nb_types
+from rbc import structure_type
+from rbc.errors import UnsupportedError, RequireLiteralValue
+from rbc.targetinfo import TargetInfo
+from rbc.typesystem import Type
+from llvmlite import ir
+
+
+class HeavyDBRowFunctionManagerType(Type):
+    """RowFunctionManager<> is a typesystem custom type that
+    represents a class type with the following public interface:
+
+      struct RowFunctionManager { }
+
+    """
+
+    @property
+    def __typesystem_type__(self):
+        ptr_t = Type.fromstring("int8 pointer")
+        return Type(ptr_t).params(NumbaPointerType=HeavyDBRowFunctionManagerNumbaType).pointer()
```

Comment:
`ptr_t == int8` seems misleading to me. Suggest 
```suggestion
        return Type('int8').params(NumbaPointerType=HeavyDBRowFunctionManagerNumbaType).pointer()
```